docs/api/upgrade-assistant/batch_reindexing.asciidoc

```diff
@@ -6,7 +6,7 @@
 
 experimental["The underlying Upgrade Assistant concepts are stable, but the APIs for managing Upgrade Assistant are experimental."]
 
-Start or resume multiple reindexing tasks in one request. Additionally, reindexing tasks started or resumed
+Start or resume multiple <<start-resume-reindex, reindexing>> tasks in one request. Additionally, reindexing tasks started or resumed
```

Comment:
This should like to the Kibana [Start or resume reindex API](https://www.elastic.co/guide/en/kibana/master/start-resume-reindex.html) page